lib/kitchen/driver/ec2.rb

```diff
@@ -194,14 +196,21 @@ def create(state) # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
           t = config[:retryable_tries] * config[:retryable_sleep]
           info "Waited #{c}/#{t}s for instance <#{state[:server_id]}> to become ready."
         end
-        server = server.wait_until(
-          :max_attempts => config[:retryable_tries],
-          :delay => config[:retryable_sleep],
-          :before_attempt => wait_log
-        ) do |s|
-          hostname = hostname(s)
-          # Euca instances often report ready before they have an IP
-          s.state.name == "running" && !hostname.nil? && hostname != "0.0.0.0"
+        begin
+          server = server.wait_until(
+            :max_attempts => config[:retryable_tries],
+            :delay => config[:retryable_sleep],
+            :before_attempt => wait_log
+          ) do |s|
+            hostname = hostname(s, config[:interface])
```

Comment:
That is true, but I would call that user error - the user has told Test Kitchen to query a parameter that is never going to be available.  A `kitchen diagnose` should reveal this.